charts/komodor-agent/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+name: komodor-agent
+description: Watches and sends kubernetes resource-related events
+icon: https://raw.githubusercontent.com/komodorio/helm-charts/master/k8s-watcher.svg
+version: 1.17.0
```

Comment:
Let's start at `1.0.0`, or even `0.1.0` until we iron out all the details

Reply:
Will change the version to 2.0.0, will be easier to see which chart a customer is using (old < 2.0.0 or new >= 2.0.0)